pkg-manifest/manifest-utils/src/getSpecFromPackageManifest.ts

```diff
@@ -3,10 +3,9 @@ import { type ProjectManifest, type DependenciesOrPeersField } from '@pnpm/types
 export function getSpecFromPackageManifest (
   manifest: Pick<ProjectManifest, DependenciesOrPeersField>,
   depName: string
-) {
+): string | undefined {
```

Comment:
This is technically now breaking but not sure I can do better here. I'm failing to figure out what the deal is, effectively what's happening is that a package depends on `ts-node`, which peer deps on `@types/node`, which exists in the repo and should be linked, but that peer dep appears as a dep of the _user_ of ts-node instead.

Reply:
is it? looks like CI is passing

Reply:
Breaking API change, I mean, in that this used to return `string` but now returns `string | undefined`.

Reply:
I guess if I'm making the other major bumps, I may as well declare this one as major too?

Reply:
Ah, no, I already got that in the PR.